src/lib.rs

```diff
@@ -39,6 +41,7 @@ pub mod ledc;
 pub mod mutex;
 pub mod peripherals;
 pub mod prelude;
+pub mod rmt;
```

Comment:
Almost forgot: the CI is failing because you need to add `#[cfg(not(feature = "riscv-ulp-hal"))]` here. :)